packages/flutter/lib/src/widgets/sensitive_content.dart

```diff
@@ -0,0 +1,311 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' show max;
+
+import 'package:flutter/services.dart' show ContentSensitivity, SensitiveContentService;
+
+import 'async.dart' show AsyncSnapshot, ConnectionState, FutureBuilder;
+import 'container.dart';
+import 'framework.dart';
+
+/// Data structure used to track the [SensitiveContent] widgets in the
+/// widget tree.
+class ContentSensitivitySetting {
```

Comment:
Could this class be private?

Reply:
By the way, I really like this class now after the recent round of changes. It has a good tight definition of what it does without needing to understand anything else.